File: app/src/main/java/com/yao/zhihudaily/model/Theme.java
package com.yao.zhihudaily.model;

import java.io.Serializable;

/**
 * Created by Administrator on 2016/9/7.
 */
public class Theme implements Serializable {

    private int id;
    private int color;
    private String name;
    private String description;
    private String thumbnail;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getColor() {
        return color;
    }

    public void setColor(int color) {
        this.color = color;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getThumbnail() {
        return thumbnail;
    }

    public void setThumbnail(String thumbnail) {
        this.thumbnail = thumbnail;
    }

    @Override
    public String toString() {
        return "Theme{" +
                "id=" + id +
                ", color=" + color +
                ", name='" + name + '\'' +
                ", description='" + description + '\'' +
                ", thumbnail='" + thumbnail + '\'' +
                '}';
    }
}

File: app/src/main/java/com/yao/zhihudaily/model/Theme.kt
package com.yao.zhihudaily.model

import java.io.Serializable

/**
 * @author Yao
 * @date 2016/9/7
 * 列表里面的Theme Item
 */
class Theme : Serializable {

    var id: Int = 0
    var color: Int = 0
    var name: String? = null
    var description: String? = null
    var thumbnail: String? = null

    override fun toString(): String {
        return "Theme{" +
                "id=" + id +
                ", color=" + color +
                ", name='" + name + '\''.toString() +
                ", description='" + description + '\''.toString() +
                ", thumbnail='" + thumbnail + '\''.toString() +
                '}'.toString()
    }
}
